docker/images/grafana_graphite/clueso-dashboard.json

```diff
@@ -1168,5 +1132,5 @@
   },
   "timezone": "",
   "title": "Clueso II",
-  "version": 15
-}
\ No newline at end of file
+  "version": 2
```

Comment:
what is this the version of?  if it's our own template, can't we use 1?

Reply:
Yes exactly, this is our own template version. I'll set it to 1.